controllers/sriovoperatorconfig_controller.go

```diff
@@ -91,17 +92,19 @@ func (r *SriovOperatorConfigReconciler) Reconcile(req ctrl.Request) (ctrl.Result
 		return reconcile.Result{}, nil
 	}
 
-	// Render and sync webhook objects
-	if err = r.syncWebhookObjs(defaultConfig); err != nil {
-		return reconcile.Result{}, err
-	}
+	if enableAdmissionController {
```

Comment:
This is also checked inside `syncWebhookObjs` function.

Reply:
`syncWebhookObjs` is only called from here so moved the check for the condition outside the `syncWebhookObjs`